docs/CONFIG-VARS.md

```diff
@@ -273,41 +272,64 @@ nfs_disk_size = 500   # 500 GB
 | server_ssl | Enable/disable SSL | string | "off" | |
 | server_ssl_cert_file | Path to the PostgreSQL SSL certificate file | string | "" | If `server_ssl` is enabled and this variable is not defined, the system default SSL certificate is used. |
 | server_ssl_key_file | Path to the PostgreSQL SSL key file | string | "" | If `server_ssl` is enabled and this variable is not defined, the system default SSL key is used. |
 | administrator_login | Admin user | string | "postgres" | |
 | administrator_password | Admin password | string | "my$up3rS3cretPassw0rd" | |
+| postgres_system_settings | Configure PostgreSQL system settings | list(object({})) | `[{ name = "max_prepared_transactions", value = "1024" }, { name = "max_connections", value = "1024" }]` | Currently `max_prepared_transactions` and `max_connections` are supported configuration values |
 
 **NOTES**:
 
 1. If you enable `server_ssl` without defining either `server_ssl_cert_file` or `server_ssl_key_file`, the system's default SSL certificate and key are used instead. By default, on Ubuntu systems we create a copy of those files and name them `ssl-cert-sas-${PG_HOST}.pem` and `ssl-cert-sas-${PG_HOST}.key`.
     - The Ansible tasks that are performed include copying the certificate and key from the PostgreSQL VM into your local workspace directory.
 2. If you are planning to use the [viya4-deployment repository](https://github.com/sassoftware/viya4-deployment) to perform a SAS Viya platform deployment where you have [full-stack TLS](https://github.com/sassoftware/viya4-deployment/blob/main/docs/CONFIG-VARS.md#tls) configured, make sure that the `V4_CFG_TLS_TRUSTED_CA_CERTS` variable in the viya4-deployment ansible-vars.yaml file points to a directory that contains the server_ssl_cert_file.
 
-Sample:
+Here is an example of the `postgres_servers` variable where the `default` entry only overrides the `administrator_password` parameter, and the `another-server` entry overrides all parameters:
 
-```bash
-# Postgres Servers
+```terraform
 postgres_servers = {
   default = {
+    administrator_password       = "D0ntL00kTh1sWay"
+    server_ip                    = "10.10.10.10"     # Assigned values for static IPs
+  },
+  another_server = {
     server_num_cpu         = 8                       # 8 CPUs
     server_memory          = 16384                   # 16 GB
     server_disk_size       = 250                     # 256 GB
-    server_ip              = ""                      # Assigned values for static IP addresses - REQUIRED
+    server_ip              = "10.10.10.11"           # Assigned values for static IPs
     server_version         = 13                      # PostgreSQL version
     server_ssl             = "off"                   # SSL flag
+    server_ssl_cert_file   = "./ssl_cert.pem"        # Path to the PostgreSQL SSL certificate file
+    server_ssl_key_file    = "./ssl_cert.key"        # Path to the PostgreSQL SSL key file
```

Comment:
Yeah I do agree that is confusing and I think I put the wrong value here in the example. The idea was that for the example `another-server` Postgres Server, the user is overriding/explicitly defining all the variables.

It would make more sense that `server_ssl` is set to `on` if we explicitly defining the cert & key file locations.
I'll update this.